### DISCUSSION TITLE
Ideas for growing Masxon series of dapps community ie the kiln and clayton

### DISCUSSION THREAD

**Discussion by GregDixonMXN** [kaiachain/kaia-dapp-mono#156] [Ideas]
What else should I do in your opinion that can help grow my community other than the purposed bounty submission.

**Comment by zxstim** [comment]
Oh man, sorry missed this. Hmm... to build a community, first is about providing value. What value you can give to others? Think about that and you can start growing.

For example, say you are good technical, build a tech community, share idea on how to build, share codes, experiences, opportunities. Over time, it will grow

**Comment by zxstim** [comment]
maybe have a weekly community call as a start? just go on stream (maybe Youtube or X) and just say your thoughts out loud.

**Comment by GregDixonMXN** [comment]
I was thinking a series of tutorials @zxstim. Would this work? I changed the title for the proposal just lmk if that works still,

**Reply by zxstim** [reply]
works too. make a proposal when you are ready.

propose one thing at a time, complete each one to claim bounty
